dbt_subprojects/daily_spellbook/models/xrpl/transfers/tokens_xrpl_amm_deposits.sql

```diff
@@ -0,0 +1,303 @@
+{{
+    config(
+        schema = 'tokens_xrpl_amm_deposits',
+        alias = 'transfers',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_date','unique_key'],
```

Comment:
```suggestion
        unique_key = ['block_month','tx_hash','evt_index'],
```

Reply:
use partition column. i'd rather move away from random unique key and use the two fields which help differentiate a row directly. the reason we use `unique_key` in evm transfers is because some of the columns can be null which breaks dbt.